src/items/static-items.md

```diff
@@ -21,6 +21,54 @@ statics:
   through a reference.
 * Constants cannot refer to statics.
 
+## Statics & generics
+
+A static item defined in a generic scope (for example in a blanket or default
+implementation) will result in exactly one static item being defined, as if
+the static definition was pulled out of the current scope into the module. 
+There will *not* be one item per monomorphization.
+
+This code:
+
+```rust
+use std::sync::atomic::{AtomicUsize, Ordering};
+
+trait Tr {
+    fn default_impl() {
+        static COUNTER: AtomicUsize = AtomicUsize::new(0);
+        println!("counter was {}", COUNTER.fetch_add(1, Ordering::Relaxed));
```

Comment:
This would make it clearer that where the message comes from.

```suggestion
        println!("default_impl: counter was {}", COUNTER.fetch_add(1, Ordering::Relaxed));
```